web/viewer.js

```diff
@@ -595,21 +595,23 @@ var PDFViewerApplication = {
 
   fallback: function pdfViewFallback(featureId) {
 //#if !(FIREFOX || MOZCENTRAL)
-//  return;
+    /* jshint -W027 */
```

Comment:
Can't we do the same `//#if !PRODUCTION` trick from below here too for consistency?